migrations/versions/840_add_application_close_date_for_framework.py

```diff
@@ -0,0 +1,31 @@
+"""Add `application_close_date` for Framework
```

Comment:
The migration should go in a separate PR from the code, so it can be released on its own.